CHANGELOG.md

```diff
@@ -2,6 +2,13 @@
 
 ---
 
+## Master
+
+* Add option to add import statements at the top of the generated swift file (for
+  storyboards) using the `import` flag. Multiple imports can be added by repeating
+  this flag.
```

Comment:
Could you add two spaces after the full-stop at the end of this line, like in other entries?

(This allows the rendered markdown to go on the next line in the same paragraph)